app/routes/quotes/components/Quote.js

```diff
@@ -82,20 +107,22 @@ export default class Quote extends Component<Props, State> {
             </div>
 
             <div className={styles.bottomRight}>
```

Comment:
You can consider moving this block as it's rather pointless (since the count of reactions is already displayed).